### ISSUE TITLE
Fix Encoding of Print ISBN in JSTOR ONIX output

### ISSUE THREAD

**Issue by vincentwj** [thoth-pub/thoth#392]
JSTOR cannot properly ingest our records due to encoding of the print ISBN: "It looks like Punctum Books is using an ISBN for PDF filenames listed in ONIX as Product Relation Code 06, which is not something the JSTOR system recognizes or stores."

To fix this, JSTOR proposes the following to be changed in the JSTOR ONIX record output:

"I think a change in the ONIX is an option: JSTOR treats Product Relation Code 13 as the PISBN in ONIX. Would it be possible to include the print ISBNs this way?:"

```
<RelatedProduct>

   <RelationCode>13</RelationCode>

   <ProductIdentifier>

      <ProductIDType>15</ProductIDType>

      <IDValue>9780000000000</IDValue>

   </ProductIdentifier>

</RelatedProduct>
```

**Comment by rhigman** [comment]
According to the [ONIX codelists](https://www.editeur.org/files/ONIX%20for%20books%20-%20code%20lists/ONIX_BookProduct_Codelists_Issue_57.html), Product Relation Code 13 indicates that the work in the record is an "Epublication based on the printed product `<RelatedProduct>`". However, our current JSTOR output includes _all_ the work's publications as `<RelatedProduct>`s, not just its printed publications, so using code 13 would not be appropriate as a general solution (as it might result in tagging e.g. EPUB editions as "printed" products).

Looking at the codes in ONIX List 51 "Product relation", the only accurate alternative seems to be Code 00 "Unspecified". @vincentwj, could you ask JSTOR whether they accept this?

**Comment by vincentwj** [comment]
Code 13 is what they proposed so the ingest doesn't stall in their system. So maybe you can just fix this for the JSTOR output? Otherwise we cannot output to them. In any case they don't ingest EPUBs so this only holds for PDFs.

**Comment by rhigman** [comment]
No, I mean if you were submitting a PDF, and the work also had an EPUB publication, the Code 13 change would claim that the PDF was an "Epublication based on" the EPUB, and therefore that the EPUB was a print product. I know this isn't an issue for punctum as you only have PDFs and print versions, but it could be for other publishers using the JSTOR output.

**Comment by vincentwj** [comment]
I don't understand this. I'd just really like our books to be properly ingested in JSTOR.

**Comment by rhigman** [comment]
Then can you please ask them whether they will accept Code 00, because Code 13 isn't appropriate.

**Comment by eshellman** [comment]
@rhigman There are many times when it's better to fit your square in a round hole than to ask for a square hole. My guess is this is one of them. You can ask for guidance from the maintainers/ implementers group https://groups.io/g/onix

**Comment by eshellman** [comment]
no chance that an onix consumer would think your epub is a print product.

**Comment by rhigman** [comment]
JSTOR confirmed via @vincentwj that the changed output works for them.
